src/main/java/jenkins/plugins/logstash/LogstashNotifier.java

```diff
@@ -107,7 +107,7 @@ private boolean perform(Run<?, ?> run, TaskListener listener) {
 
   // Method to encapsulate calls for unit-testing
   LogstashWriter getLogStashWriter(Run<?, ?> run, OutputStream errorStream, TaskListener listener) {
-    return new LogstashWriter(run, errorStream, listener, run.getCharset());
+    return new LogstashWriter(run, errorStream, listener, run.getCharset(), null);
```

Comment:
please define two constructors instead of passing null value

Reply:
OK